website/docs/reference/project-configs/require-dbt-version.md

```diff
@@ -72,19 +70,16 @@ require-dbt-version: ">=1.0.0,<2.0.0"
 </File>
 
   
-### Require a specific dbt version
-:::caution Not recommended
-With the release of major version 1.0 of dbt Core, pinning to a specific patch is discouraged.
-:::
+### Require a specific dbt version (not recommended)
```

Comment:
the 'not recommended' will become part of the header. we should either add it and use the jinja  or  keep the callout advising users it's not recommended

`### Require a specific dbt version (not recommended) {#require-a-specific-dbt-version}`
```suggestion
### Require a specific dbt version

:::info Not recommended
Pinning to a specific dbt version is discouraged because it limits project flexibility and can cause compatibility issues, especially with dbt packages. It's recommended to [pin to a major release](#pin-to-a-range), using a version range (for example, `">=1.0.0", "<2.0.0"`) for broader compatibility and to benefit from updates.
```

Reply:
example of how the (not recommended) is part of the URL
![Screenshot 2024-01-09 at 11 24 29](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/90cd1f57-5d24-44f0-af35-242be727da8a)